### ISSUE TITLE
Windows: specifying path to ImageMagick in config_defaults.py

### ISSUE THREAD

**Issue by WDBell** [Zulko/moviepy#378]
```
IMAGEMAGICK_BINARY
    For linux users, 'convert' should be fine.
    For Windows users, you must specify the path to the ImageMagick
    'convert' binary. For instance:

    IMAGEMAGICK_BINARY = r"C:\Program Files\ImageMagick-6.8.8-Q16\convert.exe"

"""

FFMPEG_BINARY = 'ffmpeg-imageio'
#~ IMAGEMAGICK_BINARY = 'auto-detect'
IMAGEMAGICK_BINARY = "C:\\Program Files\\ImageMagick-7.0.4-Q16\\magick.exe"
```

The final few lines of config_defaults.py indicate how the location of ImageMagick should be specified for use by moviepy. However, often when Image ImageMagick is installed these days (apparently with versions numbered 7x) convert.exe is **not** included. Fortunately, ImageMagick can be specified as shown here.

Suggest change to config_defaults.py.

**Comment by sephib** [comment]
Found the same solution - Please update the documentation - would have saved me 1 Hour+

**Comment by saifrahmed** [comment]
I get the error ".This error can be due to the fact that ImageMagick is not installed on your computer, or (for Windows users) that you didn't specify the path to the ImageMagick binary in file conf.py, or.that the path you specified is incorrect"

-- do we edit config_defaults.py (which comes with the distro) or conf.py -- or do we rename config_defaults.py to conf.py.  Does the file need to be in the same directory as the rest of the scripts?

**Comment by WDBell** [comment]
@saifrahmed: Are you on Windows?

**Comment by WDBell** [comment]
I'm on Windows 10. These are the last few lines of the file config_defaults.py on my computer. You may notice that ImageMagick, including its binaries, is stored in a location away from other scripts. config_defaults.py is stored in the folder C:\Python34\Lib\site-packages\moviepy.

```
IMAGEMAGICK_BINARY
    For linux users, 'convert' should be fine.
    For Windows users, you must specify the path to the ImageMagick
    'convert' binary. For instance:

    IMAGEMAGICK_BINARY = r"C:\Program Files\ImageMagick-6.8.8-Q16\convert.exe"

"""

FFMPEG_BINARY = 'ffmpeg-imageio'
#~ IMAGEMAGICK_BINARY = 'auto-detect'
IMAGEMAGICK_BINARY = "C:\\Program Files\\ImageMagick-7.0.4-Q16\\magick.exe"
```

**Comment by saifrahmed** [comment]
Thanks so much WDBell for your help!
Yes, I'm on Windows.  Important note for others: I had to restart after installing ImageMagick, otherwise I was getting missing DLL errors.
Another note, if you are using Anaconda/Conda, the revised config_defaults.py should go into your specific environment's Lib\site-packages\moviepy\.

**Comment by WDBell** [comment]
@saifrahmed, You're very welcome!

**Comment by keikoro** [comment]
Is this sth. that would still need updating in the docs? Anyone want to send a PR based on the most recent versions of the docs?

**Comment by tburrows13** [comment]
Fixed in #980.
